akka-projection-grpc/src/main/scala/akka/projection/grpc/internal/ProtoAnySerialization.scala

```diff
@@ -151,11 +157,13 @@ import scalapb.options.Scalapb
   def deserialize(scalaPbAny: ScalaPbAny): Any = {
     val typeUrl = scalaPbAny.typeUrl
     if (typeUrl == ProtoAnyTypeUrl) {
+      println(s"# deserialize 1: $typeUrl") // FIXME
       if (prefer == Prefer.Scala)
         ScalaPbAny.parseFrom(scalaPbAny.value.newCodedInput())
```

Comment:
Somewhat difficult to wrap my head around this, wouldn't this create the same as the scalaPbAny. I should look at the tests.